docs/docs/1-getting-started/4-crud-generator/2-admin-generator/2-form-generator.md

```diff
@@ -0,0 +1,488 @@
+
+| Parameter           | Type       | Required | Default     | Description                                                                                                                   |
+| ------------------- | ---------- | -------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| `name`              | `string`   | `true`   | `undefined` | The `string` that will prefix all generated `FormattedMessage` `id`s. Will be displayed as header title if no `title` is set. |
+| `title`             | `string`   | `false`  | `undefined` | The `title` that is displayed in the header of the `fieldSet`. Overrides the `name`.                                          |
+| `supportText`       | `string`   | `false`  | `undefined` | A subtitle that is shown below the `title`.                                                                                   |
+| `collapsible`       | `boolean`  | `false`  | `true`      | If `true`, allows the `fieldSet` to be collapsed.                                                                             |
+| `initiallyExpanded` | `boolean`  | `false`  | `false`     | If `true`, the `fieldSet` will be initially expanded.                                                                         |
+| `fields`            | `object[]` | `true`   | `undefined` | The list of fields that are within the `fieldSet`. Defined like any other form fields.                                        |
+
+:::caution
+`fieldSet`s cannot be nested within each other. This will result in an error.
+:::
+
+### component
+
+Arbitrary components can be used in forms as well. They are defined with an import reference:
+
+```typescript
+{
+    type: "component",
+    component: { name: "FutureProductNotice", import: "../../helpers/FutureProductNotice" },
+},
+```
+
+### optionalNestedFields
+
+Some optional fields are rarely needed.
+With `optionalNestedFields`, these can be hidden behind a switch to reduce visual clutter.
+However, all fields within `optionalNestedFields` must be combined into one type in the API.
+
+A field defined like this:
+
+```typescript
+{
+    type: "optionalNestedFields",
+    name: "dimensions",
+    checkboxLabel: "Configure dimensions",
+    fields: [
+        { type: "number", name: "width", label: "Width" },
+        { type: "number", name: "height", label: "Height" },
+        { type: "number", name: "depth", label: "Depth" },
+    ],
+},
+```
+
+has `width`, `height` and `depth` combined into one type in the entities property definition. See the example in the entity [here](https://github.com/vivid-planet/comet/blob/main/demo/api/src/products/entities/product.entity.ts#L162).
+
+:::caution
+
+-   Only optional fields work in this section. `required` or `component` fields will lead to an error.
+-   `optionalNestedFields` cannot be nested within each other. This will result in an error.
+    :::
```

Comment:
```suggestion

:::
```

This should presumably be like this.

Reply:
Nevertheless, we need to fix it. This doesn't look right 😁 

<img width="1011" alt="image" src="https://github.com/user-attachments/assets/8207c578-9f53-4c71-8ccc-c60be93249fb" />